mlflow/utils/databricks_utils.py

```diff
@@ -52,10 +53,12 @@ def is_in_databricks_notebook():
 
 
 def is_dbfs_fuse_available():
-    try:
-        return subprocess.call(["mountpoint", "/dbfs"]) == 0
-    except Exception:  # pylint: disable=broad-except
-        return False
+    with open(os.devnull, 'w') as devnull_stderr, open(os.devnull, 'w') as devnull_stdout:
```

Comment:
I think you can only open one stream and use it for both.

Do you know if open(os.devnull) works on windows? Maybe you should put it inside try catch to be sure.

Reply:
Sorry I didn't see this comment - it looks like it should work on windows: https://docs.python.org/2/library/os.html#os.devnull
https://docs.python.org/3/library/os.html#os.devnull

Reply:
I see, thanks for looking this up!